### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by ksowndar**
Infinite scrolling is added

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,7 +22,9 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem "kaminari"
+gem "bootstrap-sass"
+gem "bootstrap-kaminari-views"
```
Comment (iMagesh) [inline Gemfile:27]: do you really need these gems?

File: app/views/public/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+<script>  
+ $(window).scroll(function() {
+   var next_url = $("#infinite-scrolling a[rel='next']").attr('href');      
+   if (( next_url ) && ((window.innerHeight + window.pageYOffset) >= document.body.offsetHeight))
+   {          
+     $('#my-posts').show();
+     $.getScript(next_url);
+     return;
+   }
+ });
```
Comment (iMagesh) [inline app/views/public/index.html.erb:11]: can you move this js code into a separate js file?
keeping js code in html.erb is not a good practice

**Review by iMagesh — CHANGES_REQUESTED** [review]
I have added comments @ksowndar 
Please check and update the changes.

File: Gemfile (hunk 1 above)
Comment (ksowndar) [reply Gemfile:27]: I added these gems for views Magesh. While refreshing to load pages to show some design like "loading content/posts" I have added these gems Magesh. Need to add that Magesh. Or if designs are not needed we can remove these gems Magesh

File: app/views/public/index.html.erb (hunk 2 above)
Comment (ksowndar) [reply app/views/public/index.html.erb:11]: Ok Magesh

Comment (ksowndar) [reply app/views/public/index.html.erb:11]: > can you move this js code into a separate js file? keeping js code in html.erb is not a good practice

It is showing this error when I am trying it in a seperate file Magesh
 
image

Comment (iMagesh) [reply app/views/public/index.html.erb:11]: Figure out a way to fix it

File: Gemfile (hunk 1 above)
Comment (iMagesh) [reply Gemfile]: Okay let's have it for now.

**Comment by iMagesh** [comment]
@ksowndar is this all fixed?

**Pull request closed without merging**